R/circropper.R

```diff
@@ -5,27 +5,40 @@ library(magick)
 #' Create a new image cropped by a circle mask and leave a margin
 #'
 #' @param input_path character the image file path
-#' @param outut_path character the saved image path
-#' @param margin double
+#' @param margin double the distance between circle boundary and the original image boundary
 #'
-#' @return an image file saved in the designated folder
+#' @return a cropped image
 #' @export
+#'
 #' @examples
-#' circropper("path_to_input_img.png", "output_img.png", 20)
+#' \dontrun{circropper("bear.jpg", 20)}
```

Comment:
I have not seen this '\dontrun' notation before. Should we change the example to one that will actually function in our repo (i.e. use the milad test image?)

Reply:
That's because when I run `check()`, this example raises an error without `\dontrun`.  So I added this.

Reply:
Does this example work if you use "images/mandrill.jpg"? we should probably include examples that dont raise errors

Reply:
Yes, when I specify the correct image path, there is still an error when I run `devtool::check()`. But if you just run the function itself, it works well.  I think it doesn't mean the function is wrong because all the tests I wrote for `circropper` are working well. This will be fine.

Reply:
Oh, I just try it again, the error disappears when I change the path to "tests/milad.jpg"